src/platform/posix/tcp_ip_channel.c

```diff
@@ -244,7 +243,7 @@ static lf_ret_t TcpIpChannel_send_blocking(NetworkChannel *untyped_self, const F
           switch (errno) {
           case ETIMEDOUT:
           case ENOTCONN:
-            ssize_t bytes_written = eventfd_write(self->send_failed_event_fds, 1);
+            ssize_t bytes_written = write(self->send_failed_event_fds[1], "X", 1);
```

Comment:
does `eventfd_write` really write an "X" ?

Reply:
Actually it doesn't matter what you write you just need to write something and x is just one byte